docs/index-properties-include-1-items.md

```diff
@@ -0,0 +1,15 @@
+# Untitled string in Index Schema
+
+```txt
+https://ns.adobe.com/helix/shared/index#/properties/exclude/items
+```
```

Comment:
Isn't that the same as `docs/index-properties-exclude-items.md`?

Reply:
I don't know. the docs are autogenerated

Reply:
I clean them: https://github.com/adobe/helix-shared/blob/more-indexer-stuff/docs/index.md